packages/libs/hd-wallet/docs/decisions/0001-use-bip44.md

```diff
@@ -0,0 +1,53 @@
+# Use BIP44
```

Comment:
Maybe somewhere in this decision record, you could also mention something about not supporting `extended-public-keys`